aim/web/ui_v2/src/types/services/models/metrics/metricsAppModel.d.ts

```diff
@@ -116,3 +116,8 @@ export interface IGetPersistIndex {
 }
 
 export type groupNames = 'color' | 'style' | 'chart';
```

Comment:
@aimhubio/uidevs Let's start types names with uppercase and us type keyword at the end.
e.g.
```js
export type GroupNameType = 'color' | 'style' | 'chart';

export type GroupingSelectOptionType = { ... }; 
```